tests/GUI/spaceBasicTests.spec.ts

```diff
@@ -19,150 +19,142 @@ test.describe(
         await ApiHooks.createSpaceByName(request, newSpaceName);
         await page.goto("/");
         await page.locator("cu-web-push-notification-banner").waitFor();
+      });
 
-        await leftMenu.rightClickOnElement(newSpaceName);
-        await spaceContextMenu.clickOnOption("Rename");
-        await editSpaceNameModal.typeSpaceName(renamedSpaceName);
-        await editSpaceNameModal.clickOnSaveButton();
-        await leftMenu.assertElementIsVisible(renamedSpaceName);
-
-        await ApiHooks.deleteSpaceByName(request, renamedSpaceName);
-      }
-    );
-
-    test(
-      "Basic test for checking if creating new space with the same name is not allowed",
-      {
-        tag: "@space",
-        annotation: {
-          type: "issue",
-          description: "https://github.com/marian51/poc-playwright-typescript-gui-api/issues/33",
+      test.afterEach("Remove space after test", async ({ request }) => {
+        await ApiHooks.deleteSpaceByName(request, newSpaceName);
+      });
+
+      test(
+        "Basic test for checking if renaming existing space works correct",
+        {
+          annotation: {
+            type: "issue",
+            description: "https://github.com/marian51/poc-playwright-typescript-gui-api/issues/31",
+          },
         },
-      },
-      async ({ page, request }) => {
-        const leftMenu = new LeftMenu(page);
-        const createSpaceModal = new CreateSpaceModal(page);
-        const newSpaceName = "GUI TEST new space";
-
-        await ApiHooks.createSpaceByName(request, newSpaceName);
-        await page.goto("/");
-        await page.locator("cu-web-push-notification-banner").waitFor();
+        async ({ page }) => {
+          const spaceContextMenu = new SpaceContextMenu(page);
+          const editSpaceNameModal = new EditSpaceNameModal(page);
+          const renamedSpaceName = "RENAMED GUI TEST new space";
 
-        await leftMenu.clickOnElement("Create Space");
-        await createSpaceModal.typeSpaceName(newSpaceName);
-        await createSpaceModal.assertNameInputHasError();
-        await createSpaceModal.assertErrorMessageIsDisplayed();
-        await createSpaceModal.clickOnContinueButton();
-        await createSpaceModal.assertModalWindowIsVisible();
-
-        await ApiHooks.deleteSpaceByName(request, newSpaceName);
-      }
-    );
+          await leftMenu.rightClickOnElement(newSpaceName);
+          await spaceContextMenu.clickOnOption("Rename");
+          await editSpaceNameModal.typeSpaceName(renamedSpaceName);
+          await editSpaceNameModal.clickOnSaveButton();
+          await leftMenu.assertElementIsVisible(renamedSpaceName);
 
-    test(
-      "Basic test for checking if duplicating existing space works correct",
-      {
-        tag: "@space",
-        annotation: {
-          type: "issue",
-          description: "https://github.com/marian51/poc-playwright-typescript-gui-api/issues/48",
+          newSpaceName = renamedSpaceName;
```

Comment:
Pytanie czy nie chcemy tego zrobić analogicznie jak w taskach: usunąć rename i dodać hooki na usuwanie 2 space z `newSpaceName` oraz `renamedSpaceName` + obsługa wyjątku

Jeśli nie to taski też bym zaaktualizował, żeby było tak jak tutaj

Reply:
Byłem innego zdania, ale jednak uważam, że tak z nadpisywaniem tej zmiennej jest lepiej - jeżeli damy hooka do usuwania `newSpaceName` oraz `renamedSpaceName`, które oba zawsze by się uruchamiały po każdym teście, to prawie za każdym razem będzie się uruchamiał ten wyjątek; testu to nie zatrzyma, ale wtedy wysyłamy request gdzie z góry wiemy, że on nie przejdzie